server/backend/test/steps/send-notification.steps.ts

```diff
@@ -155,23 +166,32 @@ defineFeature(feature, (test) => {
       cameraA = camera;
       cameraA.group = new Group('group');
       cameraA.group.user = new User();
+      cameraA.group.user.expoToken = 'ExponentPushToken[000000000000]';
       await cameraRepository.persistAndFlush(cameraA);
       token = getCameraAuthToken(cameraA, keyPair.privateKey);
     });
-    and('Camera A has 1 notification', async () => {
-      cameraA.notifications.add(new Notification());
-      await cameraRepository.flush();
+    and('Camera A has 0 notifications', async () => {
+      const res = await request(app.getHttpServer())
+        .get(`/cameras/${cameraA.id}/notifications`)
+        .set('Authorization', token);
+      expect(res.status).toBe(200);
+      expect(res.body).toHaveLength(0);
     });
     when('I try to send a notification on behalf of camera A', async () => {
       sendRes = await request(app.getHttpServer())
         .put(`/cameras/${cameraA.id}/notifications`)
-        .set('Authorization', token);
+        .set('Authorization', token)
+        .send(new Notification());
     });
     then('The request succeeded', () => {
       expect(sendRes.status).toBe(200);
     });
-    and('Camera A has 2 notifications', async () => {
-      expect(cameraA.notifications.toArray()).toHaveLength(2);
+    and('Camera A has 1 notification', async () => {
+      const res = await request(app.getHttpServer())
+        .get(`/cameras/${cameraA.id}/notifications`)
+        .set('Authorization', token);
+      expect(res.status).toBe(200);
+      expect(res.body).toHaveLength(1);
     });
   });
 });
```

Comment:
I don't see any Expo SDK mocks, so I'm guessing these tests are hitting Expo's API directly? We should mock `expo-server-sdk` (or we could inject a mock `PushNotificationService` if you implement the suggestion from before) to avoid this.